packages/components/src/components/hds/toast/index.ts

```diff
@@ -0,0 +1,4 @@
+import Component from '@glimmer/component';
+import type { HdsToastSignature } from './types.ts';
+
+export default class HdsToastComponent extends Component<HdsToastSignature> {}
```

Comment:
Having a backing class is a good fallback, but considering we expect to have many other components as templates only, I'd try to get it right now so we can replicate this in the future.

You probably tried something like this and it complained with something along the lines of 'Expected 0 type arguments, but got 1.'. This, as you mentioned in the PR description, means it picks up `@types/ember__component` with a 3.x version (which seems to be required by a 3rd degree dependency in `website`, so unrelated to this package).

```suggestion
import templateOnlyComponent from '@ember/component/template-only';
import type { HdsToastSignature } from './types.ts';

const HdsToastComponent = templateOnlyComponent<HdsToastSignature>();

export default HdsToastComponent;
```

Instead of `@types/ember__component/template-only.d.ts`, it should be using `ember-source/types/stable/@ember/component/template-only.d.ts` (available in our ember version), however that's not happening and it's puzzling 🤔

Reply:
Ah that's exactly what it was. Appreciate the commit. I tried it out and it LGTM